app/controllers/api/v1/users_controller.rb

```diff
@@ -11,9 +12,19 @@ def create
     end
   end
 
+  def profile_picture
+    user = User.find_by(id: params[:id])
+  
+    if user&.profile_picture&.attached?
```

Comment:
We can have a default profile picture easily on the front end as well. Might be easier there?